pkg/storage/action_cache_asset_store.go

```diff
@@ -125,6 +139,7 @@ func (rs *actionCacheAssetStore) Get(ctx context.Context, ref *asset.AssetRefere
 	if err != nil {
 		return nil, err
 	}
+	log.Printf("Fetched from action cache")
```

Comment:
Do we want some additional useful metadata here or is this a remaining debug print statement?

Reply:
This is probably a debug statement, even if it could be adapted to provide some useful data it probably still doesn't belong in this particular PR so I'll remove it.